pkgs/os-specific/linux/busybox/default.nix

```diff
@@ -24,6 +24,10 @@ let
     }
   '';
 
+  libcConfig = lib.optionalString (useMusl || stdenv.cc.libc == "musl") ''
```

Comment:
Can do `useMusl ? stdenv.cc.libc == "musl"` in the arguments and `assert stdenv.cc.libc == "musl" -> useMusl;` down below?

Reply:
Oh and that needs to be `stdenv.hostPlatform.libc`. `stdenv.cc.libc` is the actual derivation.

Reply:
d'oh! Changed is last minute--locally I have `stdenv.isMusl` O:) which is slated to at least become `hostPlatform.isMusl` (and similar) but I haven't done so yet O:).

Will fix, thank you!

(FWIW some form of `isMusl` and `isGlibc` would prevent this sort of mistake, since it would fail to evaluate if typo or whatnot O:))

But that's not what this PR is about!